router/http/src/main/scala/io/buoyant/router/http/TracingFilter.scala

```diff
@@ -10,29 +10,30 @@ import com.twitter.finagle.tracing.Trace
 object TracingFilter extends SimpleFilter[Request, Response] {
 
   def apply(req: Request, service: Service[Request, Response]) = {
+    Trace.recordRpc(req.method.toString)
     Trace.recordBinary("http.method", req.method.toString)
```

Comment:
these request annotations won't ever conflict with response annotations, but should we namespace them anyway just so that all the request annotations have the same prefix?